### PR TITLE
Get rid of segment and call notificaion-center instead

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to  https://github.com/Giveth/giveth-dapps-v2/issues/709

 * Remove calling segment directly and instead calling notification-center
 * Remove all the things related to giveth trace

### CODE REVIEW

File: src/resolvers/projectVerificationFormResolver.ts [hunk 1]
```diff
@@ -183,19 +184,10 @@ export class ProjectVerificationFormResolver {
...
-
       await sendMailConfirmationEmail(email, project, token);
-
-      SegmentAnalyticsSingleton.getInstance().track(
-        NOTIFICATIONS_EVENT_NAMES.SEND_EMAIL_CONFIRMATION,
-        `givethId-${userId}`,
-        emailConfirmationData,
-        null,
-      );
+      await getNotificationAdapter().projectSendEmailConfirmation({
```
Comment (CarlosQ96) [inline src/resolvers/projectVerificationFormResolver.ts]: I added this before but I think we can remove it. Segment doesn't send any email as we send it with:
```
      await sendMailConfirmationEmail(email, project, token);
```

Also tracking this data is not required, and thanks for removing the callbackUrl from the analytics, that info shouldn't be exposed.

I am all for deleting code as we can speed up requests.

**Review by CarlosQ96 — APPROVED** [review]
Thanks a lot Mohammad for deleting so much code that we don't need anymore. aNd also really clean implementation.

**Comment by CarlosQ96** [comment]
I think lets merge and start testing with MaryJaf thats the best thing we can do in staging to see if anything goes wrong.

File: src/resolvers/projectVerificationFormResolver.ts [hunk 2]
```diff
@@ -183,19 +184,10 @@ export class ProjectVerificationFormResolver {
       await projectVerificationForm.save();
 
       const callbackUrl = `https://${dappUrl}/verification/${project.slug}/${token}`;
```
Comment (CarlosQ96) [inline src/resolvers/projectVerificationFormResolver.ts]: if the callback url is not used, lets remove it here. I see you build it inside the sendMailConfirmationEmail method.

File: src/resolvers/projectVerificationFormResolver.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply src/resolvers/projectVerificationFormResolver.ts]: Nice, thanks for mentioning that, I remove it from both notificaiton-center and impact-graph

**Pull request merged**